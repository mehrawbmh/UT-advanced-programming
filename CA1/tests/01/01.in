2 13:25
Ghermez 5
2
5 Panzdahe_e_Khordad 5 Khayyam
2
5 Saâ€™adi 10 Darvazeh_Dowlat
Abi 10
1
15 Hassan_Abad
2
10 Mellat 5 Baharestan
Baharestan
