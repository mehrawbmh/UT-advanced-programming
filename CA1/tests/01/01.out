Towards end of Abi in 2 station(s)
13:45
1302
